CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Added
+
+-   Implement `InputDate` component
+
+### Fixed
+
+-   Fix input value colors on `warning` and `critical` variants
+-   Fix disabled input style on Firefox
```

Comment:
Considering this is the "first" implementation, don't know if it makes sense to include a "Fixed" section for the `InputDate`

Reply:
Actually the fix for the disabled input style on Firefox was on the `useInputProps` component, we did fix the style on the other inputs 👌🏻